rotkehlchen/chain/ethereum/manager.py

```diff
@@ -310,6 +310,9 @@ def get_own_node_info(self) -> Optional[NodeName]:
                 return node
         return None
 
+    def get_connected_nodes(self) -> List[NodeName]:
+        return list(self.web3_mapping.keys())
```

Comment:
nitpick: There is really no need to have another function just for this. You can directly call `list(self.web3_mapping.keys())` wherever you need.